mkdocs.yml

```diff
@@ -4,67 +4,70 @@ nav:
-        - Skill GUI: skill_gui.md
-        - Skill API: skill_api.md
-        - Skill Json: skill_json.md
-        - F.A.Q.: dev.md
-    - GUI:
-        - Introduction: gui_intro.md
-        - Homescreen: homescreen.md
-        - QML Design Guidelines: qml_guidelines.md
-        - Open Linguistika: linguistika.md
-        - Shell: shell.md
-        - Protocol: gui_protocol.md
-        - Extensions: gui_extensions.md
-    - Plugins:
-        - Plugin Manager: OPM.md
-        - Speech To Text: stt_plugins.md
-        - Text To Speech: tts_plugins.md
-        - Wake Word: ww_plugins.md
-        - Voice Activity Detection: vad_plugins.md
-        - Grapheme to Phoneme: g2p_plugins.md
-        - Audio Playback: audio_plugins.md
-        - Language Detection/Translation: lang_plugins.md
-        - Platform/Hardware Abstraction Layer: PHAL.md
-    - Technologies:
-        - Common Play: OCP.md
-        - Personal Backend: personal_backend.md
-        - Skills Manager: osm.md
-        - Config Assistant: OCA.md
-        - STT Server: stt_server.md
-        - TTS Server: tts_server.md
-        - Translate Server: translate_server.md
-    - Images:
-        - Buildroot: buildroot.md
-        - Manjaro: manjaro.md
-    - Mycroft:
-        - Dinkum: dinkum.md
-        - Compatibility: compat.md
-        - Comparison: mycroft.md
+        - OVOS-core:
+            - Introduction: core.md
+            - Bus: bus_service.md
+            - Skills: skills_service.md
+            - Speech: speech_service.md
+            - Audio: audio_service.md
+            - GUI: gui_service.md
+        - Skills:
+            - Introduction: skills.md
+            - Design Guidelines: guidelines.md
+            - Anatomy: skill_structure.md
+            - Dialog: statements.md
+            - Prompts: prompts.md
+            - Intents: intents.md
+            - Context: context.md
+            - Converse: converse.md
+            - Filesystem: skill_filesystem.md
+            - Skill Settings: skill_settings.md
+            - Skill GUI: skill_gui.md
+            - Skill API: skill_api.md
+            - Skill Json: skill_json.md
+            - F.A.Q.: dev.md
+        - GUI:
+            - Introduction: gui_intro.md
+            - Homescreen: homescreen.md
+            - QML Design Guidelines: qml_guidelines.md
+            - Open Linguistika: linguistika.md
+            - Shell: shell.md
+            - Protocol: gui_protocol.md
+            - Extensions: gui_extensions.md
+        - Plugins:
+            - Plugin Manager: OPM.md
+            - Speech To Text: stt_plugins.md
+            - Text To Speech: tts_plugins.md
+            - Wake Word: ww_plugins.md
+            - Voice Activity Detection: vad_plugins.md
+            - Grapheme to Phoneme: g2p_plugins.md
+            - Audio Playback: audio_plugins.md
+            - Language Detection/Translation: lang_plugins.md
+            - Platform/Hardware Abstraction Layer: PHAL.md
+        - Technologies:
```

Comment:
this section should go under user

Reply:
It is to technical for the user guide, but added an advanced section to it so will move it there.

STT / TTS for the normal user guide from the setup skill, the above in the advanced section.

agree?